ConstraintLayoutPerformance/gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+android.enableJetifier=true
```

Comment:
If we switched to all `androidx` modules do we still need jetifier ?

Reply:
Shouldn't `android.enableJetifier=true ` be added to convert support libraries of potential dependencies to AndroidX?

Reply:
Not if we don't plan on adding old support libraries.